### PR TITLE
BrenKt-SK864-WEB-Email-Hosted_Exchange-firststep 

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md [hunk 1]
```diff
@@ -34,28 +40,62 @@ Pour cela, connectez-vous à ce dernier, cliquez sur `Microsoft`{.action}, puis
...
+- Si vous utilisez votre offre Exchange en complément avec un service e-mail **externe aux offres e-mail OVHcloud**, il vous sera demandé de renseigner l'URL du serveur de réception de votre service e-mails externe sous la mention `Serveur relais (SMTP)`
+
+![email](images/exchange-wizard02.png){.thumbnail}
+
+#### **Comment souhaitez-vous configurer votre zone DNS ?**
+
+- **Configuration automatique** : Le nom de domaine renseigné sera automatiquement configuré au niveau de sa Zone DNS si ce dernier est géré par OVHcloud sur le même identifiant client que votre service Exchange.
+- **Configuration manuelle** : Le nom de domaine n'est pas géré dans le même espace client que votre plateforme, il est gérer chez un autre prestataire de nom de domaine, ou vous souhaitez simplement réaliser votre configuration vous-même.<br> Vous pourrez alors retrouvez les valeurs à renseignez à la fin du processus de configuration ou dans la sections `Domaines associés`{.action} de votre plateforme
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md]:
```suggestion
- **Configuration manuelle** : Le nom de domaine n'est pas géré dans le même espace client que votre plateforme, il est géré chez un autre prestataire de nom de domaine, ou vous souhaitez simplement réaliser votre configuration vous-même.<br> Vous pourrez alors retrouver les valeurs à renseigner à la fin du processus de configuration, ou dans la section `Domaines associés`{.action} de votre plateforme.
```

File: pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md [hunk 2]
```diff
@@ -34,28 +40,62 @@ Pour cela, connectez-vous à ce dernier, cliquez sur `Microsoft`{.action}, puis
...
+### Effectuer la première configuration du service
 
 Votre service n'ayant encore jamais été utilisé, vous allez devoir réaliser sa première configuration. Celle-ci vous permettra par la suite de pouvoir utiliser vos nouvelles adresses e-mail Exchange.
 
 Pour ce faire, en accédant pour la première fois à la gestion de votre service Hosted Exchange, un assistant de configuration s'affichera. Pour débuter, cliquez sur le bouton `Commencer`{.action}.
 
-Cet assistant de configuration vous permet plusieurs manipulations. Selon votre situation, le tableau suivant peut rendre facultatives certaines étapes de ce guide.
+Cet assistant de configuration vous permet plusieurs manipulations. Selon votre situation :
+
+#### **Choisir un nom de domaine**
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md]:
```suggestion
#### Choisir un nom de domaine
```

File: pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md [hunk 3]
```diff
@@ -34,28 +40,62 @@ Pour cela, connectez-vous à ce dernier, cliquez sur `Microsoft`{.action}, puis
...
+
+Choisissez un de vos noms de domaine dans la liste ou cochez la case `Mon nom de domaine ne figure pas dans la liste ci-dessous` pour saisir un nom de domaine qui n'est pas géré dans votre espace client **mais que vous pouvez configurer**.
+
+![email](images/exchange-wizard01.png){.thumbnail}
+
+#### **Allez-vous utiliser uniquement l'offre Exchange OVH avec ce domaine ?**
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md]:
```suggestion
#### Allez-vous utiliser uniquement l'offre Exchange OVH avec ce domaine ?
```

File: pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md [hunk 4]
```diff
@@ -34,28 +40,62 @@ Pour cela, connectez-vous à ce dernier, cliquez sur `Microsoft`{.action}, puis
...
+
+La question « **Allez-vous utiliser uniquement l'offre Exchange OVH avec ce domaine ?** » va déterminer le type de configuration de votre nom de domaine. 
+- Si vous utilisez une offre Exchange seule ou avec d'autres offres **e-mail OVHcloud**, la configuration pourra se faire automatiquement ou manuellement en utilisant uniquement les serveurs e-mail d'OVHcloud.
+- Si vous utilisez votre offre Exchange en complément avec un service e-mail **externe aux offres e-mail OVHcloud**, il vous sera demandé de renseigner l'URL du serveur de réception de votre service e-mails externe sous la mention `Serveur relais (SMTP)`
+
+![email](images/exchange-wizard02.png){.thumbnail}
+
+#### **Comment souhaitez-vous configurer votre zone DNS ?**
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md]:
```suggestion
#### Comment souhaitez-vous configurer votre zone DNS ?
```

File: pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md [hunk 5]
```diff
@@ -34,28 +40,62 @@ Pour cela, connectez-vous à ce dernier, cliquez sur `Microsoft`{.action}, puis
...
+#### **Choisir un nom de domaine**
+
+Choisissez un de vos noms de domaine dans la liste ou cochez la case `Mon nom de domaine ne figure pas dans la liste ci-dessous` pour saisir un nom de domaine qui n'est pas géré dans votre espace client **mais que vous pouvez configurer**.
+
+![email](images/exchange-wizard01.png){.thumbnail}
+
+#### **Allez-vous utiliser uniquement l'offre Exchange OVH avec ce domaine ?**
+
+La question « **Allez-vous utiliser uniquement l'offre Exchange OVH avec ce domaine ?** » va déterminer le type de configuration de votre nom de domaine. 
+- Si vous utilisez une offre Exchange seule ou avec d'autres offres **e-mail OVHcloud**, la configuration pourra se faire automatiquement ou manuellement en utilisant uniquement les serveurs e-mail d'OVHcloud.
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md]:
```suggestion

- Si vous utilisez une offre Exchange seule ou avec d'autres offres **e-mail OVHcloud**, la configuration pourra se faire automatiquement ou manuellement en utilisant uniquement les serveurs e-mail d'OVHcloud.
```

File: pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md [hunk 6]
```diff
@@ -34,28 +40,62 @@ Pour cela, connectez-vous à ce dernier, cliquez sur `Microsoft`{.action}, puis
...
-|Configuration des comptes Exchange|Déterminez le nom de vos adresses e-mail Exchange et ajoutez des informations complémentaires.|
-|Migration de données (si applicable)|Si vous êtes engagé dans un processus de migration de vos adresses e-mail depuis l'une des solutions OVHcloud (MX Plan ou E-mail Pro), vous avez la possibilité d'initier cette migration depuis cet assistant. Si vous utilisez un logiciel de messagerie, il vous faudra également configurer de nouveau vos comptes.|
+#### **Cas particulier**
 
-### Étape 3 : ajouter des domaines additionnels (facultatif)
+- Si vous configurez votre plateforme Exchange avec un nom de domaine qui n'est pas géré sur le même espace client que cette plateforme ou chez un autre prestataire de nom de domaine, vous obtiendrez la fenêtre suivante :<br>
+![email](images/exchange-wizard05.png){.thumbnail .w-640}<br>
+Cette fenêtre vous invite à ajoutez un **champ CNAME** dans la zone DNS du nom de domaine. Cette entrée à pour bute de vérifier que vous gérez bien ce nom de domaine.<br>
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md]:
```suggestion
Cette fenêtre vous invite à ajouter un **champ CNAME** dans la zone DNS du nom de domaine. Cette entrée a pour but de vérifier que vous gérez bien ce nom de domaine.<br>
```

File: pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md [hunk 7]
```diff
@@ -34,28 +40,62 @@ Pour cela, connectez-vous à ce dernier, cliquez sur `Microsoft`{.action}, puis
...
+> [!warning]
+> Sans cette validation par champ CNAME, il est impossible d'utiliser la plateforme avec ce nom de domaine.
+
+- Si vous configurez votre plateforme Exchange avec un nom de domaine qui n'est pas géré sur le même espace client que cette plateforme, qu'il est géré chez un autre prestataire de nom de domaine, ou si vous avez choisissez de configurer manuellement votre nom de domaine. la fenètre suivante s'affichera :<br>
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md]:
```suggestion
- Si vous configurez votre plateforme Exchange avec un nom de domaine qui n'est pas géré sur le même espace client que cette plateforme, qu'il est géré chez un autre prestataire de nom de domaine, ou si vous avez choisi de configurer manuellement votre nom de domaine, la fenêtre suivante s'affichera :<br>
```

File: pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md [hunk 8]
```diff
@@ -34,28 +40,62 @@ Pour cela, connectez-vous à ce dernier, cliquez sur `Microsoft`{.action}, puis
...
+![email](images/exchange-wizard06.png){.thumbnail .w-640}<br>
+Vous retrouvez ici les valeurs à renseigner dans votre zone DNS. les **champs MX** qui correspondent à la réception de vos e-mails et le **champ SRV** qui correspond à la configuration automatique de vos adresses e-mail.
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md]:
```suggestion
Vous retrouvez ici les valeurs à renseigner dans votre zone DNS. Les **champs MX** correspondent à la réception de vos e-mails. Le **champ SRV** correspond à la configuration automatique de vos adresses e-mail.
```

File: pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md [hunk 9]
```diff
@@ -34,28 +40,62 @@ Pour cela, connectez-vous à ce dernier, cliquez sur `Microsoft`{.action}, puis
...
+
+Retrouvez les détails de configuration de votre zone DNS concernant votre service e-mail sur notre page [Ajouter un champ MX à la configuration de son nom de domaine](https://docs.ovh.com/fr/domains/mail-mutualise-guide-de-configuration-mx-avec-zone-dns-ovh/).
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md]:
```suggestion
Retrouvez les détails de configuration de votre zone DNS concernant votre service e-mail sur notre page « [Ajouter un champ MX à la configuration de son nom de domaine](https://docs.ovh.com/fr/domains/mail-mutualise-guide-de-configuration-mx-avec-zone-dns-ovh/) ».
```

File: pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md [hunk 10]
```diff
@@ -34,28 +40,62 @@ Pour cela, connectez-vous à ce dernier, cliquez sur `Microsoft`{.action}, puis
...
+
+### Ajouter des noms de domaines additionnels (facultatif)
 
-Une fois la première configuration de votre domaine terminée, vous pouvez également configurer des noms de domaines additionnels si vous le souhaitez et si vous ne l'avez pas déjà fait via l'assistant.
+Une fois la première configuration de votre domaine terminée, vous pouvez également configurer des noms de domaines additionnels si vous le souhaitez et si vous ne l'avez pas déjà fait, via l'assistant.
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md]:
```suggestion
Une fois la première configuration de votre domaine terminée, vous pouvez également configurer, via l'assistant, des noms de domaines additionnels si vous le souhaitez et si vous ne l'avez pas déjà fait.
```

File: pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md [hunk 11]
```diff
@@ -34,28 +40,62 @@ Pour cela, connectez-vous à ce dernier, cliquez sur `Microsoft`{.action}, puis
...
 
 > [!warning]
 >
-> Toutes les adresses créées sur votre service Exchange seront en mesure de visualiser dans l'annuaire l'ensemble des adresses de ce service, y compris celles possédant un nom de domaine différent. Pour dissocier l'affichage des domaines, vous devez commander un nouveau service Hosted Exchange pour le ou les noms de domaine concernés.
+> Toutes les adresses créées sur votre service Exchange seront visible dans l'annuaire des adresses de ce service, y compris celles possédant un nom de domaine différent. Pour dissocier l'affichage des domaines, vous devez commander un nouveau service Hosted Exchange pour le ou les noms de domaine concernés.
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md]:
```suggestion
> Toutes les adresses créées sur votre service Exchange seront visibles dans l'annuaire des adresses de ce service, y compris celles possédant un nom de domaine différent. Pour dissocier l'affichage des domaines, vous devez commander un nouveau service Hosted Exchange pour le ou les noms de domaine concernés.
```

File: pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md [hunk 12]
```diff
@@ -85,30 +124,42 @@ Les comptes en attente de configuration s'affichent dans le tableau sous la form
...
 
-OVHcloud propose des licences Outlook dans l'[espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr){.external}, ainsi que des licences Office 365 sur la page [https://www.ovhcloud.com/fr/collaborative-tools/microsoft-365/](https://www.ovhcloud.com/fr/collaborative-tools/microsoft-365/){.external}. Nous vous recommandons l'une de ces solutions si vous souhaitez bénéficier du logiciel de messagerie Outlook ou de plus de logiciels de la suite Office, selon vos besoins.
+OVHcloud propose des licences Outlook en option avec votre compte e-mail Exchange depuis l'[espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr){.external}, pour y souscrire, consultez notre page [Obtenir une licence Outlook pour Exchange](https://docs.ovh.com/fr/microsoft-collaborative-solutions/exchange-licence-outlook-exchange-2013/).
```
Comment (OVH-Team-Guides) [inline pages/web/microsoft-collaborative-solutions/exchange_starting_hosted/guide.fr-fr.md]:
```suggestion
OVHcloud propose des licences Outlook en option avec votre compte e-mail Exchange depuis l'[espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr){.external}. 
Pour y souscrire, consultez notre page « [Obtenir une licence Outlook pour Exchange](https://docs.ovh.com/fr/microsoft-collaborative-solutions/exchange-licence-outlook-exchange-2013/) ». 
```

**Review by OVH-Team-Guides — CHANGES_REQUESTED** [review]

**Review by OVH-Team-Guides — APPROVED** [review]

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
{{.cds.build.pelicanoutput}}
```

**Pull request merged**